libs/knowledge-store/notebooks/astra_support.ipynb

```diff
@@ -19,18 +19,50 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #29.        async for html in loader.alazy_load():
</pre><p>This is already simplified (the actual parsing of HTML links is extracted). I think a good next step (for future follow-up) would be a document transformer that does:</p><p> </p><ol><li>HTML to Markdown</li><li>With an optional content selector</li><li>With 0 or more "extractors to apply to beautiful soup".</li></ol><p> </p><p>This would let us replace this body with something like:</p><p> </p>

<pre class="ql-syntax" spellcheck="false">HtmlToMarkdownExtraction(
  selector = "...",
  soup_edge_extractors = [
    html_link_extractor,
  ]
)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/datastax/ragstack-ai/pull/471/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>